lib/bullhorn/orders.ex

```diff
@@ -8,8 +8,6 @@ defmodule Bullhorn.Orders do
   alias Bullhorn.Mailer
 
   def tribble_failed(%TribbleFailed{order: order, type: failure_type}) do
-    Logger.metadata(order_id: order.id)
-
```

Comment:
Any harm in leaving this in? or is it duplicated at this point? Feels like more logging is better logging

Reply:
It's a duplicate at this point. All of the `order_id` and `user_id` will be set when we first read the message. That way we can make sure any error that happens between then has the needed metadata.